packages/react/src/Footnotes/Footnotes.module.css

```diff
@@ -20,6 +20,10 @@
   padding-inline-start: var(--base-size-12);
 }
 
+.FootnotesItem {
+  scroll-margin-top: 100px;
```

Comment:
To your question:

> Are you happy with the change, and the 100px value?

Should this be different across viewports? 100px on mobile is quite a lot of offset, should it be half of that?

Reply:
It could probably be reduced a bit, but I don't know if it necessarily needs to be responsive.

Our AnchorNav, for example, is 72px tall and I'm not sure if we have any nav bars taller than that. We could bring the value down to 80px, but probably no lower than that.

Reply:
👍 sounds good, thanks